docs/getting-started-guides/scratch.md

```diff
@@ -0,0 +1,727 @@
+  - `--bip=`
+    - should be the CIDR range for pods for that specific node.
+  - `--mtu=`
+    - may be required when using Flannel, because of the extra packet size due to udp encapsulation
+  - `--insecure-registry $CLUSTER_SUBNET` 
+    - to connect to a private registry, if you set one up, without using SSL.
+
+You may want to increase the number of open files for docker:
+   - `DOCKER_NOFILE=1000000`
+
+Ensure docker is working correctly on your system before proceeding with the rest of the
+installation, by following examples given in the Docker documentation.
+
+### rkt
+
+[rkt](https://github.com/coreos/rkt) is an alterative to Docker.  You only need to install one of Docker or rkt.
+
+*TODO*: how to install and configure rkt.
+
+### kubelet
+
+All nodes should run kubelet.  See [Selecting Binaries](#selecting-binaries).
+
+Arguments to consider:
+  - If following the HTTPS security approach:
+    - `--api-servers=https://$MASTER_IP`
+    - `--kubeconfig=/var/lib/kubelet/kubeconfig`
+  - Otherwise, if taking the firewall-based security approach
+    - `--api-servers=http://$MASTER_IP`
+  - `--config=/etc/kubernetes/manifests` -%}
+  - `--cluster-dns=` to the address of the DNS server you will setup (see [Starting Addons](#starting-addons).)
+  - `--cluster-domain=` to the dns domain prefix to use for cluster DNS addresses.
+  - `--docker-root=`
+  - `--root-dir=`
+  - `--configure-cbr0=` (described above)
+  - `--register-node` (described in [Node](../node.md) documentation.
+
+### kube-proxy
+
+All nodes should run kube-proxy.  (Running kube-proxy on a "master" node is not
+strictly required, but being consistent is easier.)   Obtain a binary as described for
+kubelet. 
+
+Arguments to consider:
+  - If following the HTTPS security approach:
+    - `--api-servers=https://$MASTER_IP`
+    - `--kubeconfig=/var/lib/kube-proxy/kubeconfig`
+  - Otherwise, if taking the firewall-based security approach
+    - `--api-servers=http://$MASTER_IP`
+
+### Networking 
+Each node needs to be allocated its own CIDR range for pod networking.
+Call this $NODE_X_POD_CIDR.
+
+A bridge called `cbr0` needs to be created on each node.  The bridge is explained
+further in the [networking documentation](../networking.md).
+
+- Recommended, automatic approach:
+  1. Set `--configure-cbr0=true` option in kubelet init script and restart kubelet service.  Kubelet will configure cbr0 automatically.
+     It will wait to do this until the node controller has set Node.Spec.PodCIDR.  Since you have not setup apiserver and node controller
+     yet, the bridge will not be setup immediately.
+- Alternate, manual approach:
+  1. Set `--configure-cbr0=false` on kubelet and restart.
+  1. Create a bridge
+  - e.g. `brctl addbr cbr0`.
+  1. Set appropriate MTU
+  - `ip link set dev cbr0 mtu 1460`
+  1. Add the clusters network to the bridge (docker will go on other side of bridge).
+  - e.g. `ip addr add $CLUSTER_CIDR dev eth0`
```

Comment:
CLUSTER_CIDR is undefined in this doc.  $NODE_X_POD_CIDR is closer, but not right - you need an IP in that CIDR.

e.g. if `$NODE_X_POD_CIDR=10.9.8.0/24`, you want `10.9.8.1/24`